rest/data/host_create_test.go

```diff
@@ -81,7 +82,7 @@ func TestListHostsForTask(t *testing.T) {
 	require.NoError((&build.Build{Id: "build_1"}).Insert())
 
 	c := DBCreateHostConnector{}
-	found, err := c.ListHostsForTask("task_1")
+	found, err := c.ListHostsForTask(context.TODO(), "task_1")
```

Comment:
`context.Background()`

Reply:
I think `context.TODO()` is if you don't know what to use or don't have one yet. I don't think that's the case in a test.

> TODO returns a non-nil, empty Context. Code should use context.TODO when it's unclear which Context to use or it is not yet available (because the surrounding function has not yet been extended to accept a Context parameter).

(from https://golang.org/pkg/context/#TODO)